### PR TITLE
Release `2.0.0-beta.0`

### PR DESCRIPTION
**Pull request by uint**
For `README.md` changes, most of it is a formatter running over the file. See the rich diff for substantial stuff.

I think we've abandoned the whole automatic changelog thing, have we? If so, let's remove `scripts/update_changelog.sh` soon

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [2.0.0-beta.0] - 2024-01-23
```
Comment (chipshort) [inline CHANGELOG.md:8]: We should change the "Update to cw-std 1.5" entry to cw-std 2.0.0-beta.1

**Review by chipshort — COMMENTED** [review]
Just the changelog entry.
Yes, we abandoned the script because I could not get it to work correctly with the version branches. Agree, we should remove it.

File: CHANGELOG.md (hunk 1 above)
Comment (uint) [reply CHANGELOG.md:8]: Ahh, fair

**Review by chipshort — APPROVED** [review]
LGTM

**Comment by uint** [comment]
> Yes, we abandoned the script because I could not get it to work correctly with the version branches. Agree, we should remove it.

I'm sort of ambivalent about it. I always found it a bit quirky/fiddly. I think the big proponents of it aren't around anymore, so yeah, let's just do the same as core CosmWasm repos

**Pull request merged**